marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowRunnerImpl.java

```diff
@@ -400,8 +402,18 @@ public RunFlowResponse run(String jobId, String[] items, Map<String, Object> opt
                 }
             }
             catch(Exception e) {
-                e.printStackTrace();
-                throw new RuntimeException(e);
+                if(StringUtils.containsIgnoreCase(e.getMessage(), "Plugin error")){
```

Comment:
I recommend putting this logic into a protected method so it can be easily unit tested - can pass an Exception into the method and verify that the correct logic is executed.

Reply:
I have done that and added a test